knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -465,6 +465,17 @@ func setKafkaDeployments(replicas int32) mf.Transformer {
 			if err := unstructured.SetNestedField(u.Object, int64(replicas), "spec", "replicas"); err != nil {
 				return err
 			}
+		} else if u.GetKind() == "HorizontalPodAutoscaler" {
+			min, _, err := unstructured.NestedInt64(u.Object(), "spec", "minReplicas")
+			if err != nil {
+				return err
+			}
+			if min > replicas {
+				return nil
+			}
+			if err := unstructured.SetNestedField(u.Object(), replicas, "spec", "minReplicas"); err != nil {
```

Comment:
should `replicas` be ->   `int64(replicas)` ?

Reply:
yep, fixed the build errors and rebased :+1: